### PR TITLE
sneak_grow_overpowered modification

### PR DESCRIPTION
**Pull request by fxmorin**
Hey Gnembon,
I've been having fun learning your programming language. Decided to make one of your scripts more user-friendly.

### CODE REVIEW

File: programs/survival/sneak_grow_overpowered.sc [hunk 1]
```diff
@@ -3,7 +3,10 @@ _crouch_grow_tick() ->
     for(filter(player('*'), _ ~ 'sneaking'),
         l(x, y, z) = pos(_);
         scan(x, y, z, 8, 8, 8,
-            random_tick(_)
+            if (material(_) == 'plant',
+                particle("villagerHappy", _, 2, 0.4);
```
Comment (gnembon) [inline programs/survival/sneak_grow_overpowered.sc]: the correct name of the grow particle is 'happy_villager'

**Comment by gnembon** [comment]
Thanks. That brings it closer to what modded sneak grow does.

**Comment by fxmorin** [comment]
Right. I was thinking about maybe adding a delay to it. I’ll be trying it after exams

**Comment by fxmorin** [comment]
I also just added a delay. Should try every 4 game ticks!!

**Comment by gnembonne** [comment]
use single quotes in string :)

**Comment by fxmorin** [comment]
Is there anything else you wanted to add to it?

**Pull request merged**